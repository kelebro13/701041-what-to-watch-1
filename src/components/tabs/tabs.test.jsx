import renderer from "react-test-renderer";
import Tabs from "./tabs";

it(`renders properly`, () => {
  const wrapper = renderer
    .create(<Tabs items={[`test`]}><div>{`Test`}</div></Tabs>)
    .toJSON();

  expect(wrapper).toMatchSnapshot();
});
