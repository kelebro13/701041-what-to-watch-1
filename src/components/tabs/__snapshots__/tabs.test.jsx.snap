// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders properly 1`] = `
<div
  className="movie-card__desc"
>
  <nav
    className="movie-nav movie-card__nav"
  >
    <ul
      className="movie-nav__list"
    >
      <li
        className="movie-nav__item movie-nav__item--active"
      >
        <a
          className="movie-nav__link"
          href="#"
          onClick={[Function]}
        >
          test
        </a>
      </li>
    </ul>
  </nav>
  <div>
    Test
  </div>
</div>
`;
